src/main/java/Counter.java

```diff
@@ -0,0 +1,18 @@
+public class Counter {
+
+    private final AppLabel label;
+    private int value;
+
+    public Counter(AppLabel label) {
+        this.label = label;
+    }
+
+    public void updateValue() {
+        value += 1;
+        label.setText(Integer.toString(value));
+    }
+
+    public int getValue() {
```

Comment:
Is this method used?

Reply:
Not yet, I added it for a test which haven't included in the commit, but apparently I forgot to remove the method before committing